crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -62,7 +62,7 @@ pub struct C8yMapperConfig {
     pub bridge_service_name: String,
     pub bridge_health_topic: Topic,
     pub smartrest_use_operation_id: bool,
-    pub smartrest_child_append_device_fragment: bool,
+    pub smartrest_child_device_create_with_device_marker: bool,
```

Comment:
Not a request for change: I'm just surprised clippy isn't freaking out with that name 😉.